tesla_ver/bubble_chart.py

```diff
@@ -0,0 +1,198 @@
+import dash
+import base64
+import io
+
+from dash.dependencies import Input, Output, State
+import plotly.graph_objs as go
+
+import pandas as pd
+
+from tesla_ver.layout import LAYOUT
+from numbers import Number
+
+
+def generateBubbleChart(server):
+    app = dash.Dash(
+        __name__, server=server, url_base_pathname="/bubblechart.html/"
+    )  # , external_stylesheets=external_stylesheets)
+    app.layout = LAYOUT
+
+    def parse_contents(contents, filename, date):
```

Comment:
In the docstring, you might want to say something about what these parameters represent (contents, filename, date).

Reply:
addressed in c70fb7b

Reply:
addressed in c70fb7b